### PR TITLE
Header with emoji collections and settings pages

### PR DESCRIPTION
**Pull request by canadianveggie**
Added top nav with settings page and emoji collection.

Screen Shot 2022-01-26 at 9 13 02 PM

Emojis are displayed on either side of guesses when they unlock.
Screen Shot 2022-01-26 at 9 13 16 PM

### CODE REVIEW

**Comment by canadianveggie** [comment]
Note: I removed the emoji icon. I want to think of a better way to expose that. How hard would some kind of animation that has the emoji move through the tiles?

**Comment by wachunga** [comment]
> Note: I removed the emoji icon. I want to think of a better way to expose that. How hard would some kind of animation that has the emoji move through the tiles?

We can think of something. I don't quite know what you mean by "move through the tiles".

**Comment by wachunga** [comment]
> Not entirely happy with the extensive copy-pasting. Ideas?

How about we stick with a SPA and do some basic routing based on a "page" parameter:
https://github.com/alpinejs/alpine/issues/306#issuecomment-627400322

**Comment by canadianveggie** [comment]
> > Note: I removed the emoji icon. I want to think of a better way to expose that. How hard would some kind of animation that has the emoji move through the tiles?
> 
> We can think of something. I don't quite know what you mean by "move through the tiles".

Some kind of basic CSS animation. Whatever is easy, mobile-friendly, and kid friendly. I don't know what is possible. But something more closely tied to the word you typed in.

File: index.html [hunk 1]
```diff
@@ -119,6 +153,85 @@ <h1 class="title" x-text="'Wordle Junior ' + emoji"></h1>
...
+          </label>
+          <label>
+            <input
+              type="radio"
+              id="qwerty"
+              value="qwerty"
+              x-model="settings.keyboardLayout"
+              @change="updateSettings()"
+            />
+            QWERTY
```
Comment (wachunga) [inline index.html:231]: fancy

File: index.html [hunk 2]
```diff
@@ -129,9 +242,13 @@ <h1 class="title" x-text="'Wordle Junior ' + emoji"></h1>
     const puzzleId = getTodayPuzzleId();
     const maxGuesses = 6;
 
-    function initialKeyboard() {
+    function initialKeyboard(keyboardLayout) {
       const a_z = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("");
-      const keyboard = a_z.map((letter) => ({
+      const qwerty = "QWERTYUIOPASDFGHIJKLZXCVBNM".split("");
```
Comment (wachunga) [inline index.html:247]: "I" see a problem.
```suggestion
      const qwerty = "QWERTYUIOPASDFGHJKLZXCVBNM".split("");
```

File: public/emojis.js [hunk 3]
```diff
@@ -7,48 +7,76 @@ const emojis = {
...
   DANCER: "💃",
+  DANGO: "🍡",
   DART: "🎯",
   DEER: "🦌",
   DEVIL: "😈",
-  DINO: "🦖",
-  DINOS: "🦕🦕",
-  DOGS: "🐩🐕",
+  DINO: "🦕",
+  DODO: "🦤",
```
Comment (wachunga) [inline public/emojis.js:61]: the rarest of all

File: public/emojis.js [hunk 4]
```diff
@@ -7,48 +7,76 @@ const emojis = {
...
+  DOOR: "🚪",
   DOVE: "🕊",
+  DRESS: "👗",
+  DRUM: "🥁",
   DUCK: "🦆",
-  DUCKS: "🦆🦆",
+  EAGLE: "🦅",
   EARS: "👂",
+  EARTH: "🌎",
+  ELSA: "👸",
```
Comment (wachunga) [inline public/emojis.js:72]: Have you been getting help?

**Review by wachunga — APPROVED** [review]
Looks great1 💯

File: public/emojis.js (hunk 4 above)
Comment (canadianveggie) [reply public/emojis.js:72]: These are surprises for here. I should add OLAF too...

**Pull request merged**